README.md

```diff
@@ -282,6 +283,29 @@ would not: in all classes of browsers, it would attempt to fetch `app.mjs` direc
 <script type="module">import "./app.mjs";</script>
 ```
 
+### Feature detection
```

Comment:
This section should not be nested under "Specifier remapping examples". I think it should be a h2-level (`##`) section after "Import map processing" and before "Alternatives considered".

Reply:
Done